compiler/ir/backend.jvm/src/org/jetbrains/kotlin/backend/jvm/JvmSymbols.kt

```diff
@@ -235,7 +235,7 @@ class JvmSymbols(
     }
 
     val suspendLambdaClass: IrClassSymbol = createClass(FqName("kotlin.coroutines.jvm.internal.SuspendLambda")) { klass ->
-        klass.superTypes += suspendFunctionInterface.typeWith()
+        klass.superTypes += suspendFunctionInterface.owner.defaultType
```

Comment:
Maybe we could introduce an extension `val IrClassSymbol.defaultType: IrType` beside `starProjectedType`, to remove the `.owner.defaultType` pattern?

Reply:
Good idea, I've added it as an extension on `IrClassifierSymbol`, since the pattern appears in several other places.